include/rocksdb/c.h

```diff
@@ -630,9 +630,11 @@ extern ROCKSDB_LIBRARY_API void rocksdb_write_writebatch_wi(
     rocksdb_writebatch_wi_t* wbwi,
     char** errptr);
 extern ROCKSDB_LIBRARY_API rocksdb_iterator_t* rocksdb_writebatch_wi_create_iterator_with_base(
+    const rocksdb_readoptions_t* options,
```

Comment:
This is breaking interface. We should consider whether we want to do it, or how we should do it.

Reply:
cannot think of a way to pass ReadOptions down without requiring changing signature of `NewIteratorWithBase`.

Reply:
Can you keep the first one, and add a new function with the new option? In the old one you can create a default option and pass it in?

Reply:
sure we can do that to keep backward compatible